build_defs/internal/kotlin.bzl

```diff
@@ -4,7 +4,7 @@ Rules are re-exported in build_defs.bzl -- use those instead.
 """
 
 load("//build_defs/kotlin_native:build_defs.bzl", "kt_wasm_binary", "kt_wasm_library")
```

Comment:
Hmm maybe. Wasm builds are pretty slow though. And I don't think we have any immediate plans to actually use this code in wasm, do we?

Reply:
The wasm builds would only really get used by Travis, so it shouldn't slow down dev much...  we probably don't need it for now though, since the JS compilation provides a similar functionality in ensuring we don't use java-specific stdlib stuff.